lib/features/support/fast-selenium.rb

```diff
@@ -0,0 +1,19 @@
+# copy pasta from https://www.browserstack.com/automate/ruby
```

Comment:
Change the link to https://www.browserstack.com/automate/ruby#add-on as it isn't trivial to find the right section in the page